unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/ChannelLimitReachedWindowComponentView.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.Chat.HUD
+{
+    public class ChannelLimitReachedWindowComponentView : BaseComponentView, IChannelLimitReachedWindowView
+    {
+        [SerializeField] internal Button[] acceptButton;
```

Comment:
Lets use a button from our UI Components library (`ButtonComponentView`). If you don't need some of the features this button have, you can simply deactivate the corresponding components in the used prefab or even we could create another button prefab more simple, but it's important to use the prefabs from our UI library to keep the consistency between our huds and to make them ready to be changed from the source prefab (for example for future common style's changes).

Reply:
done!